README.rst

```diff
@@ -8,6 +8,8 @@ It is Python-based, with content in `ReStructuredText (rst) <https://docutils.so
 Running locally
 ---------------
 
+We recommend using a virtual environment like `virtualenv <https://virtualenv.pypa.io/en/latest/>`_ or `venv <https://docs.python.org/3/library/venv.html>`_. Call your virtual environment `env` or `venv`, or add you preferred name to `<.gitignore>`_.
```

Comment:
This is a really good tip and I'd like us to have it in the `README`. Let's recommend venv and our own choice of folder name, and show the command on setting it up - with  reminder to add the folder name to gitignore.